lib/redux/actions/async.js

```diff
@@ -236,6 +236,7 @@ export function doDeviceUpload(driverId, opts, utc) {
     const version = versionInfo.semver;
     const { devices, os, targetTimezones, uploadTargetUser } = getState();
     const targetDevice = _.findWhere(devices, {source: {driverId: driverId}});
+    opts = opts || {};
```

Comment:
Perhaps move this into the function signature? ES6 includes default params that are super nice/convenient: `export function doDeviceUpload(driverId, opts = {}, utc)`